### PR TITLE
Update QC with summary statistics

### PR DESCRIPTION
**Pull request by Georgehe4**
Summary for coverage rdd.

### CODE REVIEW

File: mango-python/src/mango/QC.py [hunk 1]
```diff
@@ -56,3 +56,37 @@ def CoverageDistribution(self, coverageRDD, showPlot = True):
...
+        :return: list of tuples between coverage and the number of sites covered at that coverage.
+        :rtype: (list of (string, string) tuples, double, double)
+        """
+        coverageDistribution = self.CoverageDistribution(coverageRDD, showStatistics)
+        mean = self._takeMean(coverageDistribution)
+        median = self._takeMedian(coverageDistribution)
+        if showStatistics:
+            print("Mean coverage: " + str(mean))
+            print("Median coverage: " + str(median))
+        return coverageDistribution, mean, median
```
Comment (akmorrow13) [inline mango-python/src/mango/QC.py]: what plot is this meant to generate?

File: mango-python/src/mango/QC.py [hunk 2]
```diff
@@ -31,28 +31,47 @@ def __init__(self):
...
         """
 
-    def CoverageDistribution(self, coverageRDD, showPlot = True):
-
-        # make sure coverageRDD is flattened
-        flattened = coverageRDD.flatten()
-
-        # compute coverage distribution
-        coverageDistribution = flattened.toDF().rdd \
+    def CoverageDistribution(self, coverageRDD, showPlot=True, normalize = False, cummulative = False,):
```
Comment (akmorrow13) [inline mango-python/src/mango/QC.py]: maybe we can do some sequence(CoverageRDDs) instead of CoverageRDD so we can create a plot on multiple files

**Comment by Georgehe4** [comment]
@akmorrow13 Updated QC to accept a list of coverageRDDs.

**Comment by Georgehe4** [comment]
screen shot 2017-09-11 at 11 53 57 am

Sample coverage graph.

File: mango-python/src/mango/QC.py [hunk 3]
```diff
@@ -31,28 +31,54 @@ def __init__(self):
...
+        :param bool showPlot: whether to plot the coverage distribution.
+        :param bool normalize: whether to normalize the distribution.
+        :param bool cummulative: whether to plot the cummulative distribution.
+        :return: list of coverage distributions per input coverageRDD.
+        :rtype: list of list of (string, string) tuples
+        """
+        coverageDistributions = []
+        if (not isinstance(coverageRDDs, list)):
+            coverageRDDs = [coverageRDDs]
+        for coverageRDD in coverageRDDs:
```
Comment (akmorrow13) [inline mango-python/src/mango/QC.py]: you want to combine these RDDs first
- combine RDDs (key by some identifier, ie name)
- reduce by key, where key is count and name
- run one collect

**Comment by akmorrow13** [comment]
The picture seems to be cutoff:
screen shot 2017-09-13 at 5 42 12 pm

It may be due to this output:
screen shot 2017-09-13 at 5 42 15 pm

**Pull request merged**